client/stylesheets/_toCat.scss

```diff
@@ -299,10 +299,9 @@ code {
 }
 
 .profileBtn {
-    background-color: $color-venetian-red !important;
-    border-color: $color-venetian-red !important;
+    background-color: #f8504b !important;
     &:hover {
-
+        background-color: #f73c37;
```

Comment:
Color literals like `#f73c37` should only be used in variable declarations; they should be referred to via variable everywhere else.